components/admin_console/data_retention_settings/data_retention_settings.tsx

```diff
@@ -1,5 +1,10 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/* eslint-disable react/jsx-closing-bracket-location */
```

Comment:
Could you remove this eslint-disable? I'm okay leaving the one for `max-lines` for now since I know that warning is a bit contentious within the team, but we shouldn't be disabling other rules unless we have a good reason to

Reply:
Hi,
i had to disable that rules because of a eslint bug (i suppose) because in those lines:
https://github.com/mattermost/mattermost-webapp/blob/25d3f702d5b43bc0b7c853702bfa7b492ea52b2d/components/admin_console/data_retention_settings/data_retention_settings.tsx#L578-L613

Eslint signaled an error in the closure of the component but when I went to solve it a new error appeared and when I went to solve it the first reappeared, essentially it was a vicious circle.

So the only way that i found to fix it was to disable the rule.

Reply:
Maybe we could solve that by moving that out of the final return and using a regular if statement instead of a ternary then?
```javascript
let jobTime;
if (this.state.showEditJobTime) {
    jobTime = (
        <ReactSelect
            ...
        />
    );
} else {
    jobTime = <span .../>;
}
```

Personally, I prefer to split up that sort of code when you're rendering something complicated anyway.